src/Installation/wine.md

```diff
@@ -0,0 +1,18 @@
+# Installing wine
+
+A wine version higher than 7 is required, 8.3+ is recommended.
+You can see your wine version with `wine --version`
+
+### Debian based distrobutions.
+
+See [winehq's guide](https://wiki.winehq.org/Ubuntu).
+
+### Arch based distrobutions.
+
+First, make sure to add the multilib repository and the community repository. [Official repositories](https://wiki.archlinux.org/title/Official_repositories)
+
+After that it should just be as simple as a `pacman -S wine`
```

Comment:
This documentation is extremely self deprecating and third person.

Reply:
What should i add otherwise?